CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
-### Java 11
+## 0.2.0
+
+### Known Issues
+- Ethsigner is unable to derive an appropriate nonce for a private transaction, as such the `nonce` field of `eea_SendTransaction` is mandatory - if a private transaction is submitted without a nonce an error will be returned. DApps can use the [`priv_getTransactionCount`]( (https://docs.pantheon.pegasys.tech/en/latest/Reference/Pantheon-API-Methods/#priv_gettransactioncount)) JSON RPC to determine the correct nonce prior to transaction transmission.
```

Comment:
When a private transaction is submitted without a nonce, then transaction will be rejected.
Ethsigner is unable to derive the appropriate nonce for a private transaction, as the `nonce` field of `eea_SendTransaction` is mandatory the transaction is rejected.
DApps can use the [`priv_getTransactionCount`]( (https://docs.pantheon.pegasys.tech/en/latest/Reference/Pantheon-API-Methods/#priv_gettransactioncount)) JSON RPC to determine the correct nonce prior to transaction transmission.

Reply:
Just prepend the first sentence right? Latter paragraph appears unchanged.